delta-lake/delta-spark332db/src/main/scala/com/nvidia/spark/rapids/delta/GpuDeltaParquetFileFormat.scala

```diff
@@ -51,11 +50,11 @@ object GpuDeltaParquetFileFormat {
   def tagSupportForGpuFileSourceScan(meta: SparkPlanMeta[FileSourceScanExec]): Unit = {
     val format = meta.wrapped.relation.fileFormat.asInstanceOf[DeltaParquetFileFormat]
     val requiredSchema = meta.wrapped.requiredSchema
-    if (requiredSchema.exists(_.name == IS_ROW_DELETED_COLUMN_NAME)) {
+    if (requiredSchema.exists(_.name.startsWith("_databricks_internal"))) {
```

Comment:
Curious if the value of `IS_ROW_DELETED_COLUMN_NAME` is `_databricks_internal` and if so, could we use that constant in the startsWith search?

Reply:
It's not. The open source uses a different prefix.